### ISSUE TITLE
Programmatic label for search needs to be updated

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#4068]
## Area or Component
Conduct test - person search

## Quick Summary
The programmatic label and the visual label for the search bar are not the same.

The visual label is "Search for a person to start their test" but the programmatic label is just "search"

## Accessibility Violation
Programmatic label does not convey purpose of control: https://cdc-devtools.dequecloud.com/rule-help/en/label-programmatic-not-descriptive

This is also an unequal experience for screenreader users, who aren't getting the full benefits/experience that sighted users are.

## Action Requested
Update the programmatic label to match the visual label. (Likely through `aria-labelledby`)

## Original Deque Issue
https://docs.google.com/spreadsheets/d/1b6xjubAN5Uw11DV4_B5zvSu7HeDI6Z-DpV4DMcToqAY/edit#gid=981460209&range=29:29

**Comment by BobanL** [comment]
Looking at [`aria-placeholder` documentation,](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-placeholder#description) it may seem that the `placeholder` on the `input` should be enough?
